Misc/NEWS.d/next/Library/2025-05-11-10-28-11.gh-issue-133873.H03nov.rst

```diff
@@ -0,0 +1,3 @@
+Remove the deprecated ``getmark()``, ``setmark()`` and ``getmarkers()``
+methods of the :class:`~wave.Wave_read` and :class:`~wave.Wave_write`
+classes. Patch by Bénédikt Tran.
```

Comment:
```suggestion
Remove the deprecated ``getmark()``, ``setmark()`` and ``getmarkers()``
methods of the :class:`~wave.Wave_read` and :class:`~wave.Wave_write`
classes, which were deprecated since Python 3.13. Patch by Bénédikt Tran.
```